tests/pixie/tests/test-stdlib.pxi

```diff
@@ -282,4 +282,11 @@
   (try
      (throw "This is an exception")
      (catch e
-       (t/assert= (ex-msg e) "This is an exception"))))
\ No newline at end of file
+       (t/assert= (ex-msg e) "This is an exception"))))
+
+(t/deftest test-range
+  (t/assert= (-seq (range 10))
+             (-seq (-iterator (range 10))
+             (reduce conj nil (range 10))
+             '(0 1 2 3 4 5 6 7 8 9))))
```

Comment:
`assert=` accepts/compares only 2 arguments, e.g. `(assert= 1 1 2 3 4)` wouldn't throw an exception.

I'm not sure why the additional arguments don't result in an exception. It should, I think.